### PR TITLE
Update usaco-964.mdx

### PR DESCRIPTION
**Pull request by anirudhc1229**
Added first c++ solution and first n^2 solution

_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [x] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [x] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

### CODE REVIEW

**Review by bqi343 — CHANGES_REQUESTED** [review]
Same as #2461.

Also, this is not O(N^2).

**Comment by SansPapyrus683** [comment]
lol I’d review but I’m on mobile rn

File: solutions/usaco-964.mdx [hunk 1]
```diff
@@ -80,5 +80,53 @@ print(ans, file = open('whereami.out', 'w'))
...
+#include <fstream>
+#include <string>
+#include <vector>
+#include <algorithm>
+#include <unordered_set>
+
+using namespace std;
+
+int main() {
+
```
Comment (bqi343) [inline solutions/usaco-964.mdx:99]: - No empty lines at the start and end of main
 - Tabs for indentation (as mentioned in the conventions)

File: solutions/usaco-964.mdx [hunk 2]
```diff
@@ -80,5 +80,53 @@ print(ans, file = open('whereami.out', 'w'))
 ```
 
 </PySection>
+	
+<CPPSection>
+	
+**Time Complexity:** $\mathcal{O}((N-K)^2)$
```
Comment (bqi343) [inline solutions/usaco-964.mdx:86]: should be N^3

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: solutions/usaco-964.mdx [hunk 3]
```diff
@@ -80,5 +80,48 @@ print(ans, file = open('whereami.out', 'w'))
...
+#include <algorithm>
+#include <unordered_set>
+
+using namespace std;
+
+int main() {
+    ifstream fin("whereami.in");
+    ofstream fout("whereami.out");
+
+    int N;
```
Comment (SansPapyrus683) [inline solutions/usaco-964.mdx:100]: tabs

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**